### PR TITLE
Make upgrades

### LINKED ISSUE #5: Remove usage of set-output
Running this action warns:

> The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Seems we just need to bump `@actions/core` to v1.10.0+
https://github.com/ExodusMovement/gh-release-generator/blob/60295397bf5faf10b7319984b5cde95459cdd901/package.json#L72

### PR DESCRIPTION
**Pull request by RyanZim**
- Upgrade `.nvmrc` to Node 16 to match actual GH Actions env
- Remove build minification; no benefit here
- List dependency licenses in build; technically required
- upgrade `@actions/core` to fix #5

### CODE REVIEW

**Review by nachoaIvarez — APPROVED** [review]
utACK — Out of curiosity: how did you generate the upstream licenses file?

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "src"
   ],
   "scripts": {
-    "build": "ncc build src/index.mjs -m",
+    "build": "ncc build src/index.mjs --license upstream_licenses.txt",
```
Comment (RyanZim) [inline package.json:21]: @nachoaIvarez With this option

Comment (nachoaIvarez) [reply package.json:21]: Cool. I thought that was just pointing to it not also creating one. Thanks.

**Comment by dooglio** [comment]
It's nice to see this is still being used :-) And also to see you guys in action too. Hope you're having a great day!

**Pull request merged**